### ISSUE TITLE
Feature request: `--wait` flag for `helmfile sync`

### ISSUE THREAD

**Issue by mumoshu** [roboll/helmfile#64]
Extracted from https://github.com/roboll/helmfile/pull/60#issuecomment-375730521

**Comment by mumoshu** [comment]
Do we still need this even after #230?

**Comment by embik** [comment]
I'm not sure about how helmfile operates here but do I need to set `helmDefaults.wait` for every helmfile in `helmfile.d/*` as they are processed independently? In that case a flag for `helmfile sync` would be convenient.

**Comment by mumoshu** [comment]
@embik Yep that's currently how helmfile works!

**Comment by mumoshu** [comment]
Maybe we'd better add `--force-wait` AND `--default-wait`(better name would be appreciated) for flexibility?

**Comment by jwitko** [comment]
I tried to do this very thing (use `--wait` via helmfile CLI to get it to append to every helm command).  I have a setup where for finer grained control I'm executing many separate helmfiles and I didn't want to add `wait: true` everywhere.  I ended up implementing this and I'm going to submit a PR, but finding this issue I did not implement it the way noted above.  Let me know if it needs to change but I honestly find it more expected to just have `--wait` which applies globally and takes precedence as other CLI options do.
